### PR TITLE
Update mergify.yml to work with new build matrix

### PR DESCRIPTION
**Pull request by pchiusano**
CI isn't building Ubuntu 18 anymore, so Mergify shouldn't check for it.

I've already applied ready-to-merge to this PR so I think approving it will cause Mergify to merge it's own update to the Mergify config. 🤯

### CODE REVIEW

**Review by nmcb — APPROVED** [review]
green is a go

**Comment by pchiusano** [comment]
@nmcb Thanks. :) It looks like it's gotta be a committer though, which makes sense.

**Review by hojberg — APPROVED** [review]

**Comment by pchiusano** [comment]
image

Okay looks like I gotta merge this one manually, but then after this it should work?

**Comment by aryairani** [comment]
> I've already applied ready-to-merge to this PR so I think approving it will cause Mergify to merge it's own update to the Mergify config. 🤯

I wonder why it didn't.

**Pull request merged**
